ccpp/suites_not_used/suite_FV3_HAFS_v1_thompson_noahmp.xml

```diff
@@ -18,7 +18,6 @@
       <scheme>rad_sw_pre</scheme>
```

Comment:
See comments below.

Reply:
Same response as above. Will move it out in a future PR.